### DISCUSSION TITLE
What can I save of resources and time?

### DISCUSSION THREAD

**Discussion by leonnn1** [TheLastBen/fast-stable-diffusion#1014] [Q&A]
I am trying to train o colab my first model with this dreambooth repo and After I set everything up It made the text encoder, the concept training encoder and then started the Unet, but for the Unet it will take 4 hours to finish, is there a way for me to not lose all the other data I already created so I don't need to run it again, the text encoder for the concept and the first one?

**Comment by TheLastBen** [comment]
Enable the save checkpoint feature and save the models each 2000 steps

**Reply by leonnn1** [reply]
I did that, but my question is more about, what if I want to use the same class images for other models, is there a way to reuse the training?

**Reply by TheLastBen** [reply]
if intermediary checkpoints are saved, even if the training is interrupted after that, you can load later the session and resume the training from a saved checkpoint without having to reupload the instance or concept images.

**Reply by leonnn1** [reply]
Thank you for your answer, I am still beggining with it, and it looks very amazing!
